volume/service/service_test.go

```diff
@@ -19,7 +19,7 @@ import (
 func TestServiceCreate(t *testing.T) {
 	t.Parallel()
 
-	ds := volumedrivers.NewStore(nil)
+	ds := volumedrivers.NewStore(nil, 120)
```

Comment:
Just realized that these are not ints and should be time.Duration's. Its causing the tests to timeout... pushing a fix up soon.